test/spec/widget.range.spec.js

```diff
@@ -9,6 +9,11 @@ const FORM1 =
         <input type="number" name="/widgets/range_widgets/range3" data-type-xml="int" min="0" max="5" step="1">
     </label>`;
 const FORM2 = FORM1.replace( 'step="1"', 'step="0.1"' );
+const FORM3 =
+    `<label class="question non-select or-appearance-rating">
+        <span lang="" class="question-label active">Range</span>
+        <input type="number" name="/widgets/range_widgets/range3" data-type-xml="int" min="0" max="5" step="1">
+    </label>`;
 
 runAllCommonWidgetTests( RangeWidget, FORM1, '2' );
 runAllCommonWidgetTests( RangeWidget, FORM2, '2.1' );
```

Comment:
Try running the commonWidgetTest for your template/widget combo

Reply:
tried that but it's not working since a lot of doms is missing in the rating widget. Do you have any suggestions what kind of test unit should I create?

Reply:
For now, it would be fine to just runAllCommonWidgetTests() for a few combinations of step, min, max (as in `widget.range.spec.js`). No need to add custom tests, I think. So the spec file should be just a few lines of code. If tests are failing for unknown reasons, just add the spec file to this PR and I can help. It might be a yet undiscovered bug in the widget (that's why these common tests are so fabulous).

Reply:
code updated and no error found after run the test